frontend/src/employee-frontend/components/unit/unit-details/UnitEditor.tsx

```diff
@@ -164,7 +164,7 @@ const AddressSecondRowContainer = styled.div`
   justify-content: space-between;
 
   & > :first-child {
-    margin-right: 20px;
+    margin-right: 120px;
```

Comment:
I think the correct solution would be to remove `min-width: 0` from `InputField.tsx -> Wrapper`. I broke it.